apiv2/views.py

```diff
@@ -348,8 +348,8 @@ def get(self, request,  *args, **kwargs):
         ids = results
         sound_analysis_data = get_analysis_data_for_sound_ids(request, sound_ids=ids)
         # In search queries, only include audio analyers's output if requested through the fields parameter
-        needs_analyzers_ouptut = 'analyzers_output' in search_form.cleaned_data.get('fields', '') or 'ac_analysis' in search_form.cleaned_data.get('fields', '')
```

Comment:
spelling mistake in the variable name! At least it was consistent throughout the codebase...